README.md

```diff
@@ -0,0 +1,50 @@
+Project Name
```

Comment:
- [ ] Please, use the [Readme template](https://github.com/microverseinc/readme-template) for the readme file. 🐶

Reply:
I have changed the README file as per your suggestions.